### ISSUE TITLE
Trying to use N-beats on custom dataset

### ISSUE THREAD

**Issue by BlonskiP** [jdb78/pytorch-forecasting#266]
Hello, as I'm learning to work with time series, I have founded your library. After I did your tutorial I came up with idea to train N-beats on custom dataset. Unfortunatly my results aren't so great. I would like to as you what should I change while using this model to have better results. My goal is to predict one week. 
I have two datasets, which one has a sample (136k samples) every 15-20 minutes, the other is average per day (about 2k days).
I also has features like day of week, hour, change in price etc.
Is there anything I can change, or should I use TFT as model?

Datasets:
[Small 15-20 minutes samples](https://raw.githubusercontent.com/BlonskiP/Warcraft_price_prediction/main/Token-prices-small2021-01-15.csv)
[1 day samples ](https://raw.githubusercontent.com/BlonskiP/Warcraft_price_prediction/main/Token-prices-day2021-01-15.csv) 

ColabNotebook:
[N-beats model notebook](https://colab.research.google.com/drive/1LkbfBayEReQXM87518N8APppMLEA2M0F?usp=sharing)

The one thing I'm most not sure about is max_prediction_length of 24*3*7 samples (504) with context of 720 for small periods of time.  How should I use this model more 'correctly'

**Comment by jdb78** [comment]
I had a bit better results when running the notebook. You are probably best served with a model that can handle covariates such as the TFT or DeepAR.

Side notes:
* Never automatically use the suggested learning rate as the algorithm is only 90% reliable. Give it a visual check.
* First try to overfit completely to ensure the network has the expressive power to predict your target.
* Weight decay might be too high. Adjusting the hyperparamters might be a good idea if you apply the network to a completely different dataset.
